fetch.bs

```diff
@@ -309,30 +309,29 @@ a <a>byte-case-insensitive</a> match for <var>name</var>.
   to the first such <a for=/>header</a>'s <a for=header>name</a>.
 
   <p class="note no-backref">This reuses the casing of the <a for=header>name</a> of the
-  <a for=/>header</a> already in the <a for=/>header list</a>, if any. If there are multiple matched
+  <a for=/>header</a> already in <var>list</var>, if any. If there are multiple matched
   <a for=/>headers</a> their <a for=header>names</a> will all be identical.
-  <!-- XXX Firefox and Safari adjust known header names too. -->
```

Comment:
Can you mention this removal in the commit message? Is it no longer a problem?

Reply:
`xhr/setrequestheader-case-insensitive.htm` tests this. It's a problem in Safari still, but it's filed somewhere.